prow/plugins/trigger/trigger.go

```diff
@@ -106,6 +106,7 @@ type kubeClient interface {
 	CreateProwJob(kube.ProwJob) (kube.ProwJob, error)
 }
 
+// Client represents a
```

Comment:
The suspense is killing me here! 😉

Reply:
😂

Reply:
Actually, I was stuck on this one. The only reason I think we have it is to reduce the number of arguments to several other functions as it does nothing on it own.

Reply:
I was going to write:

```go
// Client makes it easier to use functions in this package by holding references to commonly used arguments.
```

But that didn't look very useful to me.

Reply:
```golang
// Client holds the necessary structures to work with prow via logging, github, kubernetes and its configuration.
//
// TODO(fejta): consider exporting an interface rather than a struct
type Client struct {}
```